int32 servo_id
int32 target_position
int32 velocity
int32 acceleration

